### PR TITLE
Fix tipo.

### PR DESCRIPTION
**Pull request by massej**

### CODE REVIEW

File: docs/keycloak-idp.md
Comment (kkimurak) [inline docs/keycloak-idp.md:45]:
```suggestion
    - OAUTH2_GENERIC_USER_UID='preferred_username'
```

**Review by kkimurak — APPROVED** [review]
Thank you for contribution. I'm just a contributer, so please wait for maintainer (`@sachilles`) review.

**Review by sachilles — APPROVED** [review]
LGTM

**Comment by sachilles** [comment]
Thanks for your contribution.

**Pull request merged**
